source/acquaman/AMGenericStepScanController.cpp

```diff
@@ -71,88 +68,15 @@ void AMGenericStepScanController::onScanTimerUpdate()
 
 AMAction3 * AMGenericStepScanController::createInitializationActions()
 {
-	return 0;
+	return AMBeamline::bl()->createScanInitializationAction(configuration_);
 }
 
 AMAction3 * AMGenericStepScanController::createCleanupActions()
 {
-	return 0;
+	return AMBeamline::bl()->createScanCleanupAction(configuration_);
 }
 
 void AMGenericStepScanController::buildScanControllerImplementation()
 {
-	QList<AMXRFDetector *> xrfDetectors;
```

Comment:
The scan controller implementation on BioXAS contains the Zebra setup. This needs to be something that's done in BioXASBeamline.